src/TimeZones.jl

```diff
@@ -4,10 +4,12 @@ using Dates
 using Printf
 using Serialization
 using Unicode
-import Dates: TimeZone
+
+import Dates: TimeZone, UTC
```

Comment:
Lets not make more of this violation of BlueStyle?

```suggestion
import Dates: TimeZone
using: Dates: UTC
```

Reply:
How does this violate it?

Reply:
We have a guideline against using `import` ever.

Reply:
In this particular case I like the use of `import` as these are concepts that would make sense to be defined in TimeZones.jl but are currently in the Dates stdlib. By using `import` I'm explicitly calling out the special relationship here.

Reply:
Ok thats fair